### DISCUSSION TITLE
Topics for The College Guide

### DISCUSSION THREAD

**Discussion by buildwithmalik** [buildwithmalik/MjcetWiki#1] [Ideas]
## ➕ Topics ++
  We’re actively looking for new topics to include in this wiki. The more detailed it is, the more helpful it'll be.

  To get started, comment below ⬇️ with an introduction of yourself and tell us which topic you'd like to see.

**Comment by somethingdevs** [comment]
Hi, 

_Ali from final year CSE here._ I noticed the lack of research knowledge and incentive in our college and was hoping that our juniors would get the opportunity to go through those things we weren't able to. I suggest adding a section somewhere for this.
